frontend/next.config.cjs

```diff
@@ -1,7 +1,12 @@
 /** @type {import('next').NextConfig} */
+const packageInfo = require('./package.json')
```

Comment:
The comment line `/** @type....` belongs directly above the `const nextConfig` line since it provides type information for it.

```
const packageInfo = require('./package.json')

/** @type {import('next').NextConfig} */
const nextConfig = {
```